ArduPlane/Parameters.h

```diff
@@ -86,10 +86,10 @@ class Parameters {
         k_param_takeoff_throttle_min_accel,
         k_param_takeoff_heading_hold, // unused
         k_param_level_roll_limit,
-        k_param_hil_servos_unused,  // unused
-        k_param_vtail_output, // unused
-        k_param_nav_controller, // unused
-        k_param_elevon_output, // unused
+        k_param_hf_elevator_freq,
+        k_param_hf_elevator_pct,
+        k_param_hf_elev_flap_mix_gain_pct,
+        k_param_lf_elev_flap_mix_gain_pct,
```

Comment:
We can't re-use these as ancient installations may be using the storage slots in persistent storage for older values.  Effect is a user might get random values in these new parameters....